client/src/components/App/Content/LandingPage/InvestigationTable/DeskFilter/DeskFilter.tsx

```diff
@@ -32,6 +32,16 @@ const DeskFilter = ({ desks, filteredDesks, onDeskChange }: Props) => {
                         {...params}
                     />
                 }
+                renderOption={(option, { selected }) => (
+                    <>
+                        <Checkbox
+                            size='small'
+                            className={classes.optionCheckbox}
+                            checked={selected}
+                        />
+                        {option.deskName}
```

Comment:
you can use the form control label
![image](https://user-images.githubusercontent.com/52863381/102876769-5dfe0b00-444e-11eb-93c1-c2592395e763.png)

Reply:
I wrote it according to the [Autocomplete demo example](https://material-ui.com/components/autocomplete/#checkboxes).